<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Research</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>





<link rel="stylesheet" href="includes/style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">lukas lab</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li>
  <a href="team.html">
    <span class="fa fa-user-friends"></span>
     
    Team
  </a>
</li>
<li>
  <a href="research.html">
    <span class="fa fa-wrench"></span>
     
    Research
  </a>
</li>
<li>
  <a href="publications.html">
    <span class="fa fa-file-alt"></span>
     
    Publications
  </a>
</li>
<li>
  <a href="funding.html">
    <span class="fa fa-hand-holding-usd"></span>
     
    Funding
  </a>
</li>
<li>
  <a href="join.html">
    <span class="fa fa-handshake"></span>
     
    Join Us!
  </a>
</li>
<li>
  <a href="press.html">
    <span class="fa fa-bullhorn"></span>
     
    Press
  </a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="mailto:&lt;lukas.schrader@wwu.de&gt;">
    <span class="fa fa-envelope fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="http://github.com/schraderl/">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Research</h1>

</div>


<div id="section" class="section level2 tabset row">
<h2 class="tabset"></h2>
<div id="rapid-genome-evolution" class="section level3">
<h3><strong>Rapid genome evolution</strong></h3>
<div class = "col-md-8", style="text-align: justify;">
Structural variants, defined as stretches of DNA that differ in copy number (deletions, insertions and duplications), orientation (inversions) or chromosomal location (translocations) between individuals, are the largest source of inter-individual genetic variation. Such complex rearrangements can be caused by a single mutagenic event and account for most of the variation between genomes. For example in humans, any two humans will on average differ in copy number across 0.78% of their genomes!

Structural variation can have devastating fitness consequences, but there is sufficient evidence to also conclude that they provide a rich basis for adaptive evolution. Different to single nucleotide variants, structural variants are much more likely to have a large effect on an individual’s phenotype, because they affect more than a single base (see for example, the loss of pelvic appendages in freshwater stickle-back populations, Chan et al. 2010). The relatively high frequency of structural variants in natural populations, their origin from single mutagenic events and their potential for causing considerable phenotypic change make structural variants a relevant research subject for studying rapid adaptation.
We explore the origins, prevalence, and consequences of structural variation in the invasive ant <i>Cardiocondyla obscurior</i> a genomic model for eusocial evolution,  which shows considerable phenotypic variation across different introduced populations. Intriguingly, in introduced populations, genetic bottlenecks and increased inbreeding reduces efficiency of selection, while the instantaneous exposure to a novel environment opens up potential (and demand) for rapid evolutionary change. Therefore, mildly adaptive small-effect variants (e.g. SNVs) caused by mutations, retrotransposons or doublestrand-break DNA repair are more likely to be lost through drift and are thus less likely to contribute to adaptation. Hence, larger structural variants conferring larger phenotypic effects should be particularly important under this scenario. Indeed, structural variation have repeatedly been shown to confer adaptations to novel environments.
<br>
<br>
</div>
<div class = "col-md-4", style="text-align: justify;">
<img src="files/research/TE2.png" width=100%>
</div>
</div>
<div id="gaga" class="section level3">
<h3><strong>GAGA</strong></h3>
<div class = "col-md-8", style="text-align: justify;">
The Global Ant Genomics Alliance (GAGA) was initiated in 2015 mainly by Guojie Zhang (University of Copenhagen, China National GeneBank, Kunming Institute of Zoology), Jacobus (Koos) Boomsma (University of Copenhagen), and Lukas Schrader and has prospered into a global collaboration with a network of over 120 collaborators from 25 countries (visit [antgenomics.dk](antgenomics.dk) for further details). GAGA’s primary goal is to generate comprehensive genomic, transcriptomic and phenotypic resources for over 200 species selected from across the ant phylogeny. For each species, we aim to produce (1) a high-quality genome assembly (using a com- bination of PacBio, and stLFR- or HiC-sequencing), (2) gene expression data from queens and workers (using short reads), (3) microbial symbioses datasets using metagenomic 16S-rDNA sequencing, (4) (macro-)eco- logical datasets, and (5) morphological datasets from micro-CT scans. You can find more details at <a href=http://antgenomics.dk>antgenomics.dk</a>.
<br>
<br>
</div>
<div class = "col-md-4", style="text-align: justify;">
<img src="files/research/GAGA1.png" width=100%>
</div>
</div>
<div id="transposable-elements-and-genetic-innovation"
class="section level3">
<h3><strong>Transposable elements and genetic innovation</strong></h3>
<div class = "col-md-8", style="text-align: justify;">
Genome evolution is a much more dynamic process than originally presumed. Present in virtually every genome, transposable elements (TEs) are important players in this, as they actively (transposition) and passively (e.g. aberrant recombination) generate novel genetic mutations and rearrangements. TEs thus play a key role in the genetic differentiation of individuals, populations, and species. We try gaining a better understanding of TEs in genome evolution and their role as a creative force in adaptive processes. For this, we study their impact in <i>Cardiocondyla obscurior</i>, a species of invasive ants whose genome is compartmentalized in fast evolving TE-rich and slowly evolving TE-poor regions. Specifically, we address (1) whether TEs can increase adaptability in this species, (2) whether selective activity of TEs in the worker caste can generate adaptive genetic diversity, and (3) whether the HSP90-dependent cellular stress response contributes to the interplay of environment and TE activity. Our researchcombines transcriptomics, genomics, and agent-based simulations to expand our knowledge of the role of TEs in evolution and how invasive species cope with constantly changing environments.
<br>
<br>
</div>
<div class = "col-md-4", style="text-align: justify;">
<img src="files/research/TE1.png" width=100%>
</div>
</div>
<div id="canalization-and-phenotypic-robustness" class="section level3">
<h3><strong>Canalization and phenotypic robustness</strong></h3>
<div class = "col-md-8", style="text-align: justify;">
Canalization underlies the expression of steady phenotypes in the face of unsteady environmental conditions or varying genetic backgrounds. The chaperone HSP90 has been identified as a key component of the molecular machinery regulating canalization and a growing body of research suggests that HSP90 could act as a general capacitator in evolution. However, empirical data about HSP90-dependent phenotypic variation and its evolutionary impact is still scarce, particularly for non-model species.
<br>
<br>
</div>
<div class = "col-md-4", style="text-align: justify;">
<img src="files/research/canalization.png" width=50%>
</div>
</div>
<div id="phenotypic-plasticity" class="section level3">
<h3><strong>Phenotypic plasticity</strong></h3>
</div>
<div id="evolutionary-capacitators" class="section level3">
<h3><strong>Evolutionary capacitators</strong></h3>
</div>
<div id="invasion-biology" class="section level3">
<h3><strong>Invasion biology</strong></h3>
</div>
<div id="rapid-host-specialization" class="section level3">
<h3><strong>Rapid host specialization</strong></h3>
<div class = "col-md-8", style="text-align: justify;">
The ability of parasites to exploit a range of host species is a classic example of phenotypic plasticity. While such host generalism is expected to be mediated through alterations of gene expression levels, the particular molecular mechanisms of plasticity are not well understood. Aphidius ervi is a generalist parasitoid of aphids, attacking a wide range of different species. Previous research indicates that host preference and parasitoid performance vary in the absence of genetic variation and host preferences can evolve rapidly over few generations, suggesting a non-genetic mode of inheritance for host preference. We designed a laboratory natural selection experiment in the absence of genetic variation to study transcriptomic changes and the non-genetic mechanisms of inheritance that could explain the evolution of rapid adaptions to a particular host. 
This project is currently funded till 2022 as part of the DFGs SPP program on rapid adaptation (“Genetic and epigenetic mechanisms of rapid host adaptation in the aphid parasitoid Aphidius ervi”, SCHR 1554/3-1) in collaboration with researchers at the University of Münster.
<br>
<br>
</div>
<div class = "col-md-4", style="text-align: justify;">
<img src="files/research/Aervi.png" width=100%>

</div>
</div>
</div>

<p>Copyright &copy; 2021 Lukas Schrader</p>



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
