<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Team</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>





<link rel="stylesheet" href="includes/style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">lukas lab</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li>
  <a href="team.html">
    <span class="fa fa-user-friends"></span>
     
    Team
  </a>
</li>
<li>
  <a href="research.html">
    <span class="fa fa-wrench"></span>
     
    Research
  </a>
</li>
<li>
  <a href="publications.html">
    <span class="fa fa-file-alt"></span>
     
    Publications
  </a>
</li>
<li>
  <a href="funding.html">
    <span class="fa fa-hand-holding-usd"></span>
     
    Funding
  </a>
</li>
<li>
  <a href="join.html">
    <span class="fa fa-handshake"></span>
     
    Join Us!
  </a>
</li>
<li>
  <a href="press.html">
    <span class="fa fa-bullhorn"></span>
     
    Press
  </a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="mailto:&lt;lukas.schrader@wwu.de&gt;">
    <span class="fa fa-envelope fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="http://github.com/schraderl/">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Team</h1>

</div>


<div class="people">
<p class="people">
<img class="people" src="files/people/simo.png"> <b> Mohammed “Simo” Errbii </b> → Population genomics and genome evolution in ants. <br>
</p>
</div>
<br>
<div class="people">
<p class="people">
<img class="people" src="files/people/na.png"> <b> Rebecca Rothering </b> → The evolutionary significance of TEs in ants.<br> Transposable elements (TEs) impact genome evolution in many different ways. Amongst other things, they can disrupt or duplicate protein-coding genes, alter splicing patterns, and enhance or repress gene expression. Although many of this TE-introduced genetic variation is likely to be deleterious, it has become increasingly clear that TEs may also play an important role in adaptation, e.g. in the invasive ant <i>Cardiocondyla obscurior</i>. This species’ genome has a peculiar compartmentalized structure: in a genomic background containing only few TEs exist “TE islands”, regions with high TE density and enriched in fast evolving gene families (such as olfactory receptors) considered important for the colonization of novel environments. As part of the Global Ants Genomics Alliance (GAGA), many more ant genomes have been sequenced and comparative analyses on TE content and distribution will help us to better understand TE evolution in ants and answer questions like “Is the existence of TE islands unique to Cardiocondyla obscurior or do we find them in other (invasive) ant species as well?” Such analyses rely on having a good TE annotation of the genomes under study in the first place, though. Usually, this involves a very time- and labour-intensive manual curation step, which is not feasible given the large number of genomes in the GAGA project. Therefore, I am currently trying to improve upon these annotations by substituting the manual curation step with a bioinformatic pipeline. <br>
</p>
</div>
<br>
<div class="people">
<p class="people">
<img class="people" src="files/people/esther.jpg"> <b> Esther van den Boes </b> → Rapid evolution and TE-induced genetic variation in invasive ants.<br> The constant need to adapt is met differently by each individual, population, or species. In some cases, the success at adapting is obvious, e.g. the behavioral adaptation in humans, in other instances though it can be quite puzzling. Our model organism, the invasive ant species <i>Cardiocondyla obscurior</i>, successfully colonizes new habitats even though genetic variation is low. With my project, we aim to uncover how phenotypic plasticity, cryptic variation and transposable elements (TEs) interplay during this rapid evolution and how especially TEs function as genetic innovators during adaptation. <br>
</p>
</div>
<br>
<div class="people">
<p class="people">
<img class="people" src="files/people/janina.png"> <b> Janina Rinke </b> → The impact of horizontal gene transfers in ants.<br> I am interested in the evolution of social insects and particularly in understanding the genetic basis of social insect evolution and their endosymbionts. In the past, I have been investigating the mechanisms of worker policing and worker reproduction in Camponotus ants. My current research project focuses on horizontally transferred genes in ants deriving from bacteria and possible new, interesting functions of these HGTs within ants. I am using large-scale sequencing data from the Global Ant Genomics Alliance to find high-quality HGT candidates with bioinformatic methods and genomic approaches. My research tackles the question whether HGTs play a significant role in ants leading to evolutionary advances. For this, I am using RNAseq data to quantify expression of HGTs as well as molecular methods to determine the existence of high-quality HGTs. <br>
</p>
</div>
<br>
<div class="people">
<p class="people">
<img class="people" src="files/people/nic.jpg"> <b> Nic Schröder </b> → Ant virome evolution and evolutionary genomics of insect endosymbionts.<br> My main scientific interests revolve around ecology and evolution of host-microbe interactions. In the past I have been working on parasites and bacterial infections, but the focus of my research shifted towards evolutionary and environmental microbiology. I am particularly interested in the interplay between animal hosts and their bacterial symbionts, as well as in the dynamics of their microbiomes and viromes. In one of my current projects we use comparative genomics of bacterial insect endosymbionts to shed light on the evolutionary transition of free-living bacteria to an endosymbiotic lifestyle. My most recent project aims to characterize the viromes of 150+ ant species using a metagenomic approach on large-scale sequencing data as part of the Global Ant Genomics Alliance. <br>
</p>
</div>
<br>
<div class="people">
<p class="people">
<img class="people" src="files/people/maja.jpg"> <b> Maja Drakula </b> → Colony recognition and self-/non-self discrimination in invasive ants. <br>
</p>
</div>
<br>
<div class="people">
<p class="people">
<img class="people" src="files/people/nadine.jpg"> <b> Nadine Kortüm </b> → Molecular characterization of ant LGTs. <br>
</p>
</div>
<br>
<div class="people">
<p class="people">
<img class="people" src="files/people/lukas.png"> <b> Lukas Schrader </b><br> My goal as a researcher is understanding the interplay between “hard-wired” genomic traits, environmental cues, “soft-wired” regulatory mechanisms, and the phenotype. <br>
</p>
</div>
<p><br><br></p>
<p><strong>Former Members:</strong></p>
<ul>
<li><strong>Lena Wiegert:</strong> Canalization and phenotypic diversification in <em>Cardiocondyla obscurior</em></li>
<li><strong>Svenja Kieckhäfer:</strong> The role of cuticular hydrocarbons in desiccation resistance in ants</li>
<li><strong>Eyleen Wiese:</strong> The genetic basis of CHC biosynthesis in ants</li>
<li><strong>Miles Winter:</strong> HSP90/HSP70 in <em>Cardiocondyla obscurior</em></li>
</ul>

<p>Copyright &copy; 2021 Lukas Schrader</p>



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
